MyBlog.Services/Areas/Users/UserService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.EntityFrameworkCore;
+using MyBlog.Infrastructure;
+using MyBlog.Infrastructure.Entities;
+using MyBlog.Service.Exception;
+
+namespace MyBlog.Service.Areas.Users;
+
+public class UserService : IUserService
+{
+    private readonly MyBlogContext _db;
+
+    public UserService(MyBlogContext db)
+    {
+        _db = db;
+    }
+
+    public async Task<List<User>> GetAllAsync()
+    {
+        return await _db.Users.ToListAsync();
+    }
+
+    public async Task<User> GetByIdAsync(int id)
+    {
+        return await _db.Users.FirstOrDefaultAsync(u => u.Id == id)
+               ?? throw new NotFoundException($"User with Id: {id} is not found");
+    }
+
+    public async Task CreateAsync(User userInput)
+    {
+        await _db.Users.AddAsync(userInput);
+    }
```

Comment:
Are you sure, that this method will complete successfully ?
Did you test this ?

If it goes wrong, don`t forget after some changes in your dbContext write next:
await _db.SaveChangesAsync();

Be focus and responsible in your tasks